woocommerce/woocommerce-ios
# Duplicate Info.plist, adding special keys for Stripe Terminal to Debug-Info.plist

Closes #3593 

Second attempt at adding the keys required by the Stripe Terminal SDK to Info.plist.

See #3656  and the discussion in the comments. That discussion justifies the changes to this PR.

Also see p1613141952407700-slack-mobile-woo

## Changes
* Duplicated the Info.plist file. 
* Assigned Debug-Info.plist to Debug builds and Release-Info.plist to release builds
* Added the keys required by the Stripe Terminal SDK to Debug-Info.plist: https://stripe.com/docs/terminal/sdk/ios#configure

## Testing
* 👀 on the Info.plist file, in particular the one for release builds.
* Remove the app from the simulator.
* Checkout the branch, build and run on the simulator.
* Check that the app is installed, and it displayed the proper name.


cc: @allendav  and @AliSoftware 

Update release notes:
